design/ordinal-crm.Rmd

```diff
@@ -0,0 +1,453 @@
+---
+title: "Design of ordinal CRM"
+author: "John Kirkpatrick and Wojciech Wójciak"
+date: "Last run on `r Sys.Date()` by `r Sys.info()[['user']]` on `r Sys.info()['nodename']`"
+output:
+  pdf_document:
+    number_sections: true
+  html_document:
+    number_sections: true
+---
+
+```{r setup, include=FALSE}
+library(ggplot2)
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# The statistical model and theory about it.
+
+The patient's toxicity response is not binary (no DLT, DLT) but categorical (for example: no DLT,
+sub-DLT, DLT). Furthermore, multicategory toxicity responses are treated as ordinal due to their
+ordinal characteristics such as a monotone trend.
+
+## The model: Cumulative Link Models
+
+These type of responses can be modeled using Generalized Linear Model (GLM), where the model is
+expressed in terms of the cumulative response probabilities. Hence, the name: Cumulative Link Model,
+or CLM. The CLM assumes the form: $$
+h[p_{ij}] = h[P(Y_i \leq j)] = \alpha_j + \mathbf x_i^T \boldsymbol \beta, \qquad j = 1, \ldots, c - 1; \, i \geq 1
+$$ where $c \in \{2, 3, \ldots \}$ denotes a number of toxicity categories on ordinal scale, and the
```

Comment:
mention what i is? I guess patient index

Reply:
Yes, `i` indexes the participant.  `Yi` is the level of toxicity reported by the ith participant (`Yi`` in {0, ..., (c-1)}).  `xi` is the dose administered to the ith participant.  See bullet 2 below this equation.